server/game/cards/06-WoE/RevoHooligans.js

```diff
@@ -0,0 +1,34 @@
+const Card = require('../../Card.js');
+
+class RevoHooligans extends Card {
+    // After Reap: Put a paint counter on an artifact. While that
+    // artifact has a paint counter, it belongs to house Mars (instead
+    // of its other houses).
+    setupCardAbilities(ability) {
+        this.reap({
+            effect: 'put a paint counter on {0}, making it belong to house Mars',
+            target: {
+                cardType: 'artifact',
+                gameAction: [
+                    ability.actions.addPaintCounter(),
+                    ability.actions.cardLastingEffect((context) => ({
```

Comment:
I think this should be a persistentEffect with a conditional clause.

Reply:
I'm having trouble figuring out how to give a persistent effect to another card within a game action.  Could you help me figure out what this should look like exactly?  I can't find any examples.

Reply:
but the effect is on itself.

Reply:
I don't understand.  Revo is putting an effect on an artifact, via the paint counter.  The effect will survive even if Revo dies.  I don't know of any other card in the game that does this.  Could you tell me which functions to call to make this a persistent effect, rather than the way I coded it?